azimuth/plots/dataset_warnings.py

```diff
@@ -554,82 +512,72 @@ def create_histogram_mean_std(
     fig = fig_default(fig)
-    fig.update_layout(margin=dict(l=60))  # The tick labels are smaller for this plot
+    fig.update_layout(margin=dict(l=60))  # Overwriting default for this plot
+
     return cast(PlotSpecification, json.loads(fig.to_json()))
 
 
 def word_count_plot(
-    train_per_class_count: np.ndarray,
-    eval_per_class_count: np.ndarray,
-    threshold_mean: float,
-    threshold_std: float,
-    train_per_class_stats: np.ndarray,
-    eval_per_class_stats: np.ndarray,
-    train_overall_stats: Tuple[np.ndarray, np.ndarray],
-    eval_overall_stats: Tuple[np.ndarray, np.ndarray],
-    divergence_mean: np.ndarray,
-    divergence_std: np.ndarray,
+    hist_per_label_per_split: Dict[DatasetSplitName, np.ndarray],
+    agg_per_split: Dict[DatasetSplitName, Dict[Agg, float]],
+    agg_per_label_per_split: Dict[DatasetSplitName, Dict[Agg, np.ndarray]],
+    divergence_per_label_per_agg: Dict[Agg, np.ndarray],
     class_names: List[str],
 ) -> DatasetWarningPlots:
     """Create the plot with the dropdown for all labels.
 
     Args:
-        train_per_class_count: Training set word count per class.
-        eval_per_class_count: Evaluation set word count per class.
-        threshold_mean: threshold above which there is an alert for the mean.
-        threshold_std: threshold above which there is an alert for the std dev.
-        train_per_class_stats: Mean and std dev per class on the train.
-        eval_per_class_stats: Mean and std dev per class on the evaluation set.
-        train_overall_stats: Mean and std dev overall on the train.
-        eval_overall_stats: Mean and std dev overall on the evaluation set.
-        divergence_mean: Alert value for the mean.
-        divergence_std: Alert value for the std dev.
+        hist_per_label_per_split: Histogram of word count per label per split.
+        agg_per_split: Mean and std dev per split.
+        agg_per_label_per_split: Mean and std dev per label per split.
+        divergence_per_label_per_agg: Alert value for the mean and std dev per label.
         class_names: List of class names.
 
     Returns:
         Plot for all classes combined, and plot per class.
 
     """
-    # Sanitize values
-    train_per_class_count, eval_per_class_count = map(
-        lambda data: np.nan_to_num(np.array(data, dtype=float)),
-        (train_per_class_count, eval_per_class_count),
-    )
-    train_per_class_stats, eval_per_class_stats = map(
-        lambda data: np.nan_to_num(np.array(data, dtype=float)),
-        (train_per_class_stats, eval_per_class_stats),
-    )
+    hist_per_split = dict()
+    for split in hist_per_label_per_split.keys():
+        # Sanitize values
+        agg_per_label = agg_per_label_per_split[split]
+        for agg in agg_per_label.keys():
+            agg_per_label[agg] = np.nan_to_num(agg_per_label[agg])
+
+        hist_per_split[split] = hist_per_label_per_split[split].sum(axis=0)
```

Comment:
In a similar fashion as before, instead of creating an empty `dict()` and then adding stuff to it in a loop, we can do all that in a dict comprehension:
```diff
-    hist_per_split = dict()
```
```suggestion
        hist_per_split = {split: h.sum(axis=0) for split, h in hist_per_label_per_split.items()}
```